### DISCUSSION TITLE
pseudo 3d conception

### DISCUSSION THREAD

**Discussion by quadrixx** [mapsforge/vtm#1076] [Q&A] [answered]
Hey there, i am new to VTM and got a question to ask. Does vtm generate third dimension automatically, based on the Attribute table(for example Z axis for the building), or map, i wanna work with, should be somehow defined as 3d map?

**Comment by devemux86** [comment, marked as answer]
It uses the OpenStreetMap tags, see the section in the [tag-mapping](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-map-writer/src/main/config/tag-mapping.xml#L420).
Then probably need to check them in the code to see how they work.

**Reply by quadrixx** [reply]
Thanks for the reply!!! So, am i right, that its possible to add a 3d feature to the map, which is actually floor plan?
